README.md

```diff
@@ -87,11 +88,20 @@ git clone --recursive https://github.com/Eppie-io/Eppie-App.git eppie-app
 
 #### To Build and Launch
 
-1. Open the **src/Eppie.App/Eppie.App.sln** file in the Visual Studio
-2. Set the **Eppie.App.UWP** project as your starting project
-3. Select **x64** platform
-4. Start building the solution (**Visual Studio Menu**: **Build** &#10148; **Build Solution**)
-5. Launch the **Eppie.App.UWP** project (**Visual Studio Menu**: **Debug** &#10148; **Start Debugging**)
+1. Open the `src/Eppie.App/Eppie.App.sln` file in Visual Studio.
+2. Set the appropriate project as your startup project:
+   - **UWP**: Select `Eppie.App.UWP`.
+   - **Windows App SDK**: Select `Eppie.App`.
+3. Select the **x64** platform.
+   - For **Windows App SDK**, also set the target framework to `net9.0-windows10.0.19041`.
```

Comment:
now version is `net9.0-windows10.0.22621`

Reply:
done